pkg/controller/datavolume/upload-controller.go

```diff
@@ -196,19 +196,37 @@ func isPVCUploadPopulation(pvc *corev1.PersistentVolumeClaim) bool {
 	return populators.IsPVCDataSourceRefKind(pvc, cdiv1.VolumeUploadSourceRef)
 }
 
+func (r *UploadReconciler) shouldUpdateStatusPhase(pvc *corev1.PersistentVolumeClaim, dv *cdiv1.DataVolume) (bool, error) {
```

Comment:
Is this function duplicated in import controller?

Reply:
There's common code but one function uses `isPVCImportPopulation` and `AnnImportPod` and the other the upload equivalent. I can try to have one shared function but I think this practice is somewhat common in our controllers.